CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ Changes are grouped as follows
 - Support for the `/simulators/models` and `/simulators/models/revisions` API endpoints.
 - Support for the `/simulators` and `/simulators/integration` API endpoints.
 
+
+## [7.73.8] - 2025-03-21
+### Fixed
+- Removed `"py38"` from list of valid runtimes for Functions in docstrings and type annotations. Runtime `"py38"` was deprecated in the API in October 2024.
```

Comment:
```suggestion
- Functions: Removed `"py38"` from list of valid runtimes for Functions in docstrings and type annotations. Runtime `"py38"` was removed in the API in October 2024.
```
Was the runtime deprecated (but still functioning) or removed? Or was it deprecated in Octover 24 and removed... later?